src/components/Home.jsx

```diff
@@ -1,7 +1,44 @@
 import React from 'react';
+import getToken from '../lib/tokens';
+import { Redirect } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
 
 const Home = () => {
-  return <h1>Home</h1>;
+  // const [hasToken, setHasToken] = useState(false);
+
+  let history = useHistory();
+
+  const Token = function generateToken() {
+    const token = getToken();
+    localStorage.setItem('token', token);
+    history.push('/list-view');
+
+    console.log(token);
```

Comment:
Is this left over from debugging? From a security/privacy perspective, we'd want to be sure that we are never logging tokens or any similar _secret_ data. It's okay for debugging locally, but is best practice to not make it into production.

Reply:
Absolutely, has been removed -thanks for this Kevin